api/instance/v1/instance_utils_test.go

```diff
@@ -28,93 +28,127 @@ func TestAttachIP(t *testing.T) {
 		reverse      = "1.1.1.1"
 	)
 
-	// Create server
-	createServerResponse, err := instanceAPI.CreateServer(&CreateServerRequest{
-		Zone:         zone,
-		Name:         "instance_utils_test",
-		Organization: organization,
-		Image:        image,
+	t.Run("create server", func(t *testing.T) {
```

Comment:
why is this test in `TestAttachIP` ?

Reply:
I reused added it to the test scenario (to reuse setup/teardown). I should be change the name of the function.